storage/tests/python/src/norcow.py

```diff
@@ -4,29 +4,55 @@
 from . import consts
 
 
-def align4_int(i: int):
-    return (4 - i) % 4
+def align_int(i: int, align: int):
+    return (align - i) % align
```

Comment:
This is weird. I'd just do `(-i) % align`.

Reply:
fixed https://github.com/trezor/trezor-firmware/commit/7b195900fe382ee4c713e131499d3eaadcb10235